### PR TITLE
Adding in mobile font sizes

### PR DESCRIPTION
**Pull request by iryanclarke**
I also didn't realize we needed this as well! I can't seem to find out how to put these under the master objects in Figma tokens, so just leaving as a base size for now

### CODE REVIEW

File: data/transformed-tokens.json [hunk 1]
```diff
@@ -577,6 +577,40 @@
...
+      },
+      "h5": {
+        "value": "16px",
+        "type": "fontSizes"
+      },
+      "body": {
+        "value": "16px",
+        "type": "fontSizes"
+      },
+      "footnode": {
```
Comment (alexandra-lim) [inline data/transformed-tokens.json:606]:
```suggestion
      "footnote": {
```

**Review by alexandra-lim — COMMENTED** [review]
How do you feel about not defining a base size for h4 and onwards? Currently only h1-h3 have typography and line height changes at mobile sizes.

We can structure the main typography tokens to fall under `desktop` and `mobile` and use the base values to define them, but we may have to consider coming up with new names for `size.h1`, etc. since it wouldn't be mapped 100% to the main tokens.

**Comment by iryanclarke** [comment]
@alexandra-lim I had thought about it, and I'm totally open to it, I figured for consistency sake we should keep the 1 to 1 mapping of the values. And obviously consistency in the case that these values **do** change for mobile, but I don't think thats likely to happen

File: data/tokens.json [hunk 2]
```diff
@@ -410,6 +410,40 @@
...
+        },
+        "h5": {
+          "value": "16px",
+          "type": "fontSizes"
+        },
+        "body": {
+          "value": "16px",
+          "type": "fontSizes"
+        },
+        "footnode": {
```
Comment (alexandra-lim) [inline data/tokens.json:439]: Sorry I commented on the wrong file! `transformed-tokens.json` is auto-generated 😅 

```suggestion
        "footnote": {
```

Comment (iryanclarke) [reply data/tokens.json:439]: 😂

**Comment by iryanclarke** [comment]
@alexandra-lim @CCAyl Removed everything from H5 down!

**Pull request merged**